firstvoices/backend/management/commands/copy_site.py

```diff
@@ -0,0 +1,581 @@
+        category.last_modified_by = user
+        category.save()
+
+        for child_category in child_categories:
+            child_category.parent = category
+            child_category.created_by = user
+            child_category.last_modified_by = user
+            child_category.save()
+
+    # Updated category list, copying over rest of categories
+    target_site_copied_over_categories = list(
+        Category.objects.filter(site=target_site).values_list("title", flat=True)
+    )
+    categories = list(
+        Category.objects.filter(site=source_site)
+        .exclude(title__in=target_site_copied_over_categories)
+        .distinct()
+    )
+    for category in categories:
+        source_category_id = category.id
+        target_category_id = uuid.uuid4()
+        category_map[source_category_id] = target_category_id
+
+        category.id = target_category_id
+        category.site = target_site
+        category.created_by = user
+        category.last_modified_by = user
+        category.save()
+
+    return category_map
+
+
+def copy_audio_and_speakers_and_return_map(source_site, target_site, logger):
+    audio_map = {}
+
+    audio_files = list(Audio.objects.filter(site=source_site))
+    for audio_file in audio_files:
+        try:
+            target_file = File(
+                content=UploadedFile(audio_file.original.content.file),
+                site=target_site,
+            )
+            target_file.save()
+            audio_file.original = target_file
+
+            # Speakers
+            current_speakers = list(audio_file.speakers.all())
+            updated_speakers = []
+            for person in current_speakers:
+                person.id = uuid.uuid4()
+                person.site = target_site
+                person.save()
+                updated_speakers.append(person)
+
+            audio_file.site = target_site
+
+            source_audio_id = audio_file.id
+            target_audio_id = uuid.uuid4()
+            audio_map[source_audio_id] = target_audio_id
+
+            audio_file.id = target_audio_id
+            # To circumvent checks added to media models to prevent modification of original file
+            audio_file._state.adding = True
+            audio_file.save()
+
+            audio_file.speakers.set(updated_speakers)
+        except Exception as e:
+            logger.warning(
+                f"Couldn't copy audio file with id: {audio_file.id}", exc_info=e
+            )
+
+    # Copy over rest of the people who are not attached as speakers to any audio
+    person_list = list(Person.objects.filter(site=source_site, audio_set__isnull=True))
+    for person in person_list:
+        person.id = uuid.uuid4()
+        person.site = target_site
+        person.save()
+
+    return audio_map
+
+
+def copy_images_and_return_map(source_site, target_site, logger):
+    image_map = {}
+
+    image_files = list(Image.objects.filter(site=source_site))
+    for image_file in image_files:
```

Comment:
[minor] same as above, for clarity I think "image_file" should be "image_model" so they don't get confused with the content of an image model